src/lib.rs

```diff
@@ -126,14 +129,17 @@ pub fn get_compact_symbol_table(
     debug_data: &WasmMemBuffer,
     breakpad_id: &str,
     dest: &mut CompactSymbolTable,
-) -> bool {
+) -> JsValue {
```

Comment:
I have reconsidered my position on the `dest` outparameter. It turns out my worries were unwarranted: wasm_bindgen has great support for returning types of the form `Result<SomeRustStruct, JsValue>`, so we should make use of that support.
Specifically: Can you please remove the `dest` parameter, and instead make this function return a value of type `Result<CompactSymbolTable, JsValue>`? Then we won't even need to use `JsValue::NULL` for the error case - when there is no error, we'll just return `Ok(table)`. You can probably even use `.map_err` here to turn the the `GetSymbolsError` into a `JsValue` (via a `GetSymbolsErrorJson`).